api/migrations/0043_order_latitude_order_longitude_alter_order_bond_size.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.5 on 2023-10-03 20:12
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0042_alter_order_logs_alter_robot_avatar"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="order",
+            name="latitude",
+            field=models.DecimalField(
+                decimal_places=6,
+                max_digits=8,
+                null=True,
+                validators=[
+                    django.core.validators.MinValueValidator(-90),
+                    django.core.validators.MaxValueValidator(90),
+                ],
+            ),
+        ),
+        migrations.AddField(
+            model_name="order",
+            name="longitude",
+            field=models.DecimalField(
+                decimal_places=6,
+                max_digits=9,
+                null=True,
+                validators=[
+                    django.core.validators.MinValueValidator(-180),
+                    django.core.validators.MaxValueValidator(180),
+                ],
+            ),
+        ),
+        migrations.AlterField(
+            model_name="order",
+            name="bond_size",
+            field=models.DecimalField(
+                decimal_places=2,
+                default=3.0,
+                max_digits=4,
+                validators=[
+                    django.core.validators.MinValueValidator(1.0),
+                    django.core.validators.MaxValueValidator(15.0),
+                ],
+            ),
+        ),
```

Comment:
This was generated together with `makemigrations`, not sure if it was just missing or a mistake on my side

Reply:
It is not a mistake, but it is an undesired change. It is best to delete lines L39 to L51 before merging (that is `migrations.AlterField(...)` . Alternatively, you can also delete the migration file, edit `.env` to `MIN_BOND_SIZE = 2` (instead of current value of 1) and the run `manage.py makemigrations` again. Should yield the same migration without altering the field `bond_size`.

The reason why this happened is that `/api/models/order.py` Order field `bond_size` depends on two config parameters that are read from the `.env` file. In my setup the `.env` file defines the `MIN_BOND_SIZE` as 2%, while the `.env-sample` defines it as 1% (at some point, there was a deviation, I changed my own `.env` without updating the `.env-sample`...). This migration will therefore allow bond sizes to be smaller than the current minimum of 2%.

I have opened a new issue (https://github.com/RoboSats/robosats/issues/873) so we make sure backend models are not dependant on any coordinator specific setting (`.env`) and therefore, we do not have any sort of conflict at run time when coordinators change values on their `.env` file.